Atomic/diff.py

```diff
@@ -8,6 +8,12 @@
 from . import Atomic
 
 class Diff(Atomic):
+
+    def diff_tty(self):
```

Comment:
I'm not in love with how this is formulated.  The main use case for diff is to use it from the CLI; which you therefore expect something on stdout.  But in the code, we can diff_tty and then diff.  Can we reverse the logic there and call diff_dbus (instead of tty) and go from there?